azure/template.json

```
@@ -251,6 +257,10 @@
                             {
                                 "name": "SENTRY_DSN",
                                 "value": "[parameters('sentryDSN')]"
+                            },
+                            {
+                                "name": "SENTRY_DSN",
```

```suggestion
                                "name": "FIND_BASE_URL",
```

Case of the copy and paste!